### DISCUSSION TITLE
More control over providers sort order

### DISCUSSION THREAD

**Discussion by KalAvroniev** [umbrellaplug/umbrellaplug.github.io#328] [Q&A] [answered]
I've been trying to figure out if I can sort the providers in a specific order.
I have RD, EasyNews and PlexShare enabled.
What I'm trying to achieve is to have the order as:
1. PlexShare
2. RD
3. EasyNews

I have set the debrid priority of RD to 5 and providers priority of PlexShare to 1 and EasyNews of 10
I tried Prioritize Direct to top but that gives me:
1. PlexShare
2. EasyNews
3. RD

I guess those priorities are separate for debrid and provider sources.

If anyone knows how to achieve the above it would be appreciated. If not then this could be an enhancement suggestion.

**Comment by Omarscomin** [comment, marked as answer]
Have you tried:

Plex Share-priority 8
RD-priority 9 
EasyNews-priority 10

Make sure to clear provider cache after making adjustments.

**Reply by KalAvroniev** [reply]
That worked! Thank you
